datadog_checks_base/datadog_checks/checks/prometheus/mixins.py

```diff
@@ -474,7 +474,7 @@ def poll(self, endpoint, pFormat=PrometheusFormat.PROTOBUF, headers=None):
             disable_warnings(InsecureRequestWarning)
             verify = False
         try:
-            response = requests.get(endpoint, headers=headers, stream=True, timeout=10, cert=cert, verify=verify)
+            response = requests.get(endpoint, headers=headers, stream=False, timeout=10, cert=cert, verify=verify)
```

Comment:
nit: the param already defaults to `False` so you can just remove `stream` altogether.